scripts/TPI.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import scipy.optimize as opt
+
```

Comment:
As you are using functions from `households.py` , ... import them
e.g. 
import households as hh
import firm